contrib/utilities/automatic_launch/generate_cases_locally.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
I would maybe add some prints here and there saying that everything is working, like "Cases generated successfully!" or maybe print the list of generated cases. Something that would give the user the feeling the thing is working as expected.